doc/book/helpers/placeholder.md

```diff
@@ -131,6 +131,21 @@ foreach ($this->data as $datum): ?>
 <?= $this->placeholder('foo')->data ?>
 ```
 
+## Clearing Content
+
+In certain situations it is desirable to remove or clear containers and aggregated content. The placeholder view helper
+provides two methods to either delete a specific container or clear all containers at once:
+
+```php
+<?php
+// Delete a single Container
```

Comment:
Please split this code example. One example for deleting a container and a second example to clear all containers.

> \### Delete a single container
>
> \```php
> $this->plugin('placeholder')->deleteContainer('myNamedContainer');
> \```
>
> \### Clear all containers
>
> \```php
> $this->plugin('placeholder')->clearContainers();
> \```